Content.Server/GameObjects/EntitySystems/Atmos/GasTileOverlaySystem.cs

```diff
@@ -161,11 +162,12 @@ private bool TryRefreshTile(GridAtmosphereComponent gam, GasOverlayData oldTile,
             }
 
             var tileData = new List<GasData>();
+            var atmosSystem = Get<AtmosphereSystem>();
```

Comment:
Can you just store this as a member variable on GasTileOverlaySystem? TryRefreshTile is called a lot.